lib/screens/explore_screens/invite/interactor/viewmodels/invite_state.dart

```diff
@@ -3,15 +3,15 @@ part of 'invite_bloc.dart';
 class InviteState extends Equatable {
   final PageState pageState;
   final PageCommand? pageCommand;
-  final String? errorMessage;
+  final InviteError? errorMessage;
   final RatesState ratesState;
   final bool isAutoFocus;
   final TokenDataModel tokenAmount;
   final FiatDataModel fiatAmount;
   final TokenDataModel? availableBalance;
   final FiatDataModel? availableBalanceFiat;
   final bool isCreateInviteButtonEnabled;
-  final String? alertMessage;
+  final InviteError? alertMessage;
```

Comment:
`alertMessage` -> `error` 🤓

Reply:
On another note and something to check, do we need 2 properties for `InviteError` in `InviteState`?

Reply:
one is used on a snakbar and the other one on a label, I think if we merge them it could lead to the wrong error being displayed.

Reply:
If they are different errors and we still need 2 separate properties, maybe we can name them better? Something like `alertError`? But something here doesn't smell right 😅 @gguijarro-c-chwy We could use some of your architectural wisdom here 🤓

Reply:
IMO it would be good to have 2 enums because one is error **from conditions of the screen** and another is general error **Exepciones, httpError**